data/category-ir-scholar

```diff
@@ -3,7 +3,6 @@
 ac.ir
 edu.ir
 iau.ac.ir
-iranlms.ir
```

Comment:
Why removed this domain?

Reply:
> Why removed this domain?

I thought iranlms.ir was one with iranlms.org.

Reply:
@rootmelo92118 please update your opinion on this PR. Should we give it a go?